### ISSUE TITLE
Support users with javascript turned off

### ISSUE THREAD

**Issue by georgiamoon** [newamericafoundation/newamerica-cms#1180]
We should have a good fallback condition for users that have javascript turned off. I know this isn't super common, but it probably maps to more accessibility/803 compliance issues as well (which we should confirm/look into).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/113920320650537/193461866861709)

**Comment by LomaxRx** [comment]
We're now using react/js for dynamic content, so this may be more [complexity then I can manage](http://jamesknelson.com/universal-react-youre-doing-it-wrong/), since it would require something like [python-react](https://github.com/markfinger/python-react) and separate set of components to be rendered by the server.

Worth noting: The homepages and post pages (excluding the Weekly and Reports) will work without javascript. I think for now I'm going to settle with a polite "Sorry, this page requires javascript. If you're concerned about us tracking you, please [read our privacy policy](https://www.newamerica.org/policies-and-procedures/). In short, we tastefully use cookies to improve site speed, and use Google Analytics to track page views and clicks."
